docs/runbooks/VirtualMachineCRCErrors.md

```diff
@@ -14,26 +14,28 @@ major service outages.
 
 ## Diagnosis
 
-1) Get the volume name from the VM:
+Execute the following PromQL expression to get a list of Virtual Machines that
+are using a storage class that is incorrectly configured:
 
-```bash
-$ kubectl get vm <vm-name> -o jsonpath='{.spec.template.spec.volumes}'
+<!--DS: You can use the Openshift metrics explorer available at 'https://{OPENSHIFT_BASE_URL}/monitoring/query-browser'.-->
+
+```promql
+kubevirt_ssp_vm_rbd_volume{rxbounce_enabled="false", volume_mode="Block"} == 1
 ```
 
-2) Get the storage class name from the volume:
+|                            | name               | namespace | pv_name            | rxbounce_enabled | volume_mode | value |
+|----------------------------|--------------------|-----------|--------------------|------------------|-------------|-------|
+| kubevirt_ssp_vm_rbd_volume | testvmi-gwgdqp22k7 | test-ns   | testvmi-gwgdqp22k7 | false            | Block       | 1     |
 
-```bash
-$ kubectl get pvc <volume-name> -o jsonpath='{.spec.storageClassName}'
-```
+You will see a list of Virtual Machines that are using a storage class without
```

Comment:
Suggestion: "The output displays a list of VirtualMachines that use a storage class without `rxbounce_enabled`.